src/gui/systray.h

```diff
@@ -76,9 +76,9 @@ class Systray
     void openHelp();
     void shutdown();
 
-    Q_INVOKABLE void hideWindow();
```

Comment:
how did you check that they are not needed ?

Reply:
Those are signals. They are exposed automatically. https://doc.qt.io/qt-5/qtqml-cppintegration-exposecppattributes.html#exposing-signals

Reply:
I vaguely remember seeing those as warnings reported by clazy
I was asking to know if you had used a tool to do that

Reply:
I just saw them and then recalled that we talked about it. But yes I think too that clazy reports them.

Reply:
My idea was just to leave the code a bit cleaner than it was before I touched code in that area.